pennylane_lightning/lightning_kokkos/_state_vector.py

```diff
@@ -26,6 +52,392 @@ class LightningKokkosStateVector:  # pylint: disable=too-few-public-methods
         dtype: Datatypes for state-vector representation. Must be one of
             ``np.complex64`` or ``np.complex128``. Default is ``np.complex128``
         device_name(string): state vector device name. Options: ["lightning.kokkos"]
+        kokkos_args(InitializationSettings): binding for Kokkos::InitializationSettings
+            (threading parameters).
+        sync(bool): immediately sync with host-sv after applying operations
```

Comment:
How is 'sync' utilized to LK advantage now?

Reply:
I am not sure if still used for something but I am afraid to break something in the future. I think that I will leave this for the moment and check if still need it before merging the `backend` branch to `master`.